apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -1,182 +1,198 @@
-      }
-
-      .min {
-        min-height: $size-min;
-        width: $size-min;
-        top: calc(50% - #{$half_min});
-        left: calc(50% - #{$half_min});
-      }
-
-      .hours {
-        min-height: $size-hours;
-        width: $size-hours;
-        top: calc(50% - #{$half_hours});
-        left: calc(50% - #{$half_hours});
-      }
-    }
+  // we use vh since we are dealing with a square and the smaller dimension is the height
+  grid-template-columns: calc(var(--clock-size) + 5vw) 1fr;
+  gap: 2.5vw;
+  grid-template-areas: 'clock schedule';
+  text-transform: uppercase;
+}
 
-    .studio-timer {
-      color: var(--studio-active, $red-active);
-      font-size: calc(#{$clock-size} / 3);
-      margin-top: calc(50% - calc(#{$clock-size} / 7));
-      line-height: 0.8em;
+.clock-container {
+  grid-area: clock;
+  display: grid;
+  place-content: center;
+  justify-items: center;
+  padding-top: 3em;
+  gap: 0.5em;
+  width: 100%;
+  height: 100%;
+  position: relative;
+  font-family: seven-seg;
+}
 
-      &--with-seconds {
-        font-size: calc(#{$clock-size} / 4.6);
-        margin-top: calc(50% - calc(#{$clock-size} / 11));
-      }
-    }
+.clock-indicators {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
 
+  .min,
+  .hours {
+    border-radius: 50%;
+    position: absolute;
+    background: var(--studio-idle, $red-idle);
 
-    .next-title:after,
-    .next-title:after,
-    .next-countdown--overtime:after {
-      content: '\200b';
+    &--active {
+      background: var(--studio-active, $red-active);
     }
+  }
 
-    .next-title {
-      color: var(--studio-idle-label, $cyan-idle);
-      text-align: center;
-    }
+  .min {
+    min-height: $size-min;
+    width: $size-min;
+    top: calc(50% - #{$half_min});
+    left: calc(50% - #{$half_min});
+  }
 
-    .next-countdown {
-      color: var(--studio-active-label, $cyan-active);
+  .hours {
+    min-height: $size-hours;
+    width: $size-hours;
+    top: calc(50% - #{$half_hours});
+    left: calc(50% - #{$half_hours});
+  }
+}
 
-      font-size: 10vh;
-      line-height: 1em;
+.studio-timer {
+  font-size: calc(var(--clock-size) / 5);
+  line-height: 1em;
 
-      &--overtime {
-        color: darken($red-active, 10%);
-      }
-    }
+  color: var(--studio-active, $red-active);
 
-    .next-countdown::before {
-      content: '-';
-    }
+  &--with-seconds {
+    font-size: calc(var(--clock-size) / 6.5);
   }
 
-  /* ============= SCHEDULE STUFF =================*/
+  &::before {
+    content: '88:88';
+    mix-blend-mode: hard-light;
+    opacity: 0.1;
+    position: absolute;
+  }
+}
 
-  .schedule-container {
-    grid-area: schd;
-    margin: 4vh 0;
-    font-family: digital-clock, monospace;
-    text-transform: uppercase;
+.next-title {
+  font-family: monospace;
+  font-weight: 400;
+  color: var(--studio-active-label, $cyan-active);
+  font-size: calc(var(--clock-size) / 10);
+  text-align: center;
+  overflow: clip visible;
```

Comment:
should we change this to 
`overflow: clip hidden;` just to catch the edge case

![image](https://github.com/cpvalente/ontime/assets/19875125/9a7b5af8-c283-4c57-836f-469d1fcae1d3)